src/Rector/MethodCall/WebTestCaseAssertIsSuccessfulRector.php

```diff
@@ -99,7 +100,7 @@ public function refactor(Node $node): ?Node
         $newArgs = [];
         // When we had a custom message argument we want to add it to the new assert.
         if (isset($args[2])) {
-            if ($args[2]->value instanceof FuncCall) {
+            if ($args[2]->value instanceof FuncCall || $args[2]->value instanceof Variable || $args[2]->value instanceof Node\Expr\Cast\String_) {
```

Comment:
I guess it can be any dynamic Expr? 

```
$this->exprAnalyzer->isDynamicExpr($args[2]->value))

Reply:
@samsonasik agreed updated the code and removed unused use statements.